nerdlets/shared/nrql-factory/index.js

```diff
@@ -354,14 +356,16 @@ class SPAFactory extends NrqlFactory {
             applicationId
             servingApmApplicationId
           }
-          relationships {
-            source {
-              entity {
-                domain
-                guid
-                type
-                ... on ApmApplicationEntityOutline {
-                  alertSeverity
+          relatedEntities {
+            results {
```

Comment:
since this went from relationships.source to relatedEntities.results i assume code elsewhere needs to be updated when reading it?

Reply:
Yes indeedly good call